editor/store/effects.js

```diff
@@ -324,8 +324,11 @@ export default {
 
 		dispatch( savePost() );
 	},
-	SETUP_EDITOR( action ) {
-		const { post, settings } = action;
+	SETUP_EDITOR( action, { getState } ) {
+		const { post } = action;
```

Comment:
I think settings was part of the setup editor action just because of this effect. And now we are not using it anymore or passing it, so In my opinion, the settings parameter should be removed from setupEditor action.